examples/with-apollo/yarn.lock

```diff
@@ -0,0 +1,3163 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Merging this in. Will create a new PR to remove all of them inside the examples folder.